dockerfiles/ci-testing/Dockerfile

```diff
@@ -1,8 +1,8 @@
-# moveit/moveit2:foxy-ci-testing
+# ghcr.io/ros-planning/moveit2:${ROS_DISTRO}-ci-testing
```

Comment:
What's the benefit of switching the container registry?
Does that require changing all our docs that refer to docker containers?

Reply:
> Does that require changing all our docs that refer to docker containers?

For MoveIt 2 we don't actually have any documentation on dockers. For MoveIt, I think we only refer them in the website, which I will send a PR, for sure to update docker location.